src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,61 @@
-export const PeopleFilters = () => {
+/* eslint-disable no-restricted-syntax */
+import React from 'react';
+import classNames from 'classnames';
+import { Link, useSearchParams } from 'react-router-dom';
+import { Filter } from '../types/Filter';
+import { getSearchWith } from '../utils/searchHelper';
+
+export const PeopleFilters: React.FC = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const sex = searchParams.get(Filter.SEX) || null;
+  const name = searchParams.get(Filter.NAME) || '';
+  const centuries = searchParams.getAll('centuries') || null;
```

Comment:
why do you need to set false value as fallback? you will get falsy value anyway if you don't find a param
```suggestion
  const sex = searchParams.get(Filter.SEX);
  const name = searchParams.get(Filter.NAME);
  const centuries = searchParams.getAll('centuries');
```